pages/pay-grades.md

```diff
@@ -2,14 +2,61 @@
+Find out more about the GS system from the [Office of Personnel Management (OPM)] (https://www.opm.gov/policy-data-oversight/pay-leave/pay-systems/general-schedule/).
+
+### Compensation in the GS pay system
+GS grade levels specify a fixed compensation range for a particular position within the federal government. This is also true for 18F, so understanding the relationship between GS grade level and compensation is important when considering applying to our organization.
+
+**Please note:** the annual salary cap for all federal employees is $158,700. You cannot be offered more than this under any circumstance.
+
+New team members can currently join 18F at GS 13, GS 14 and GS 15. Here are the base compensation ranges for each grade level taken from the [OPM SALARY TABLE 2015-GS](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/salary-tables/15Tables/html/GS.aspx):
+- Grade 13 – $73,115 to $95,048
+- Grade 14 – $86,399 to $112,319
+- Grade 15 – $101,630 to $132,122
+
+**Those ranges are the base rate only; they do not include adjustments for locality pay.** 
+
+Locality pay is the way in which the GS pay system accounts for the variation in cost of living. An example of locality for Washington DC and San Francisco is illustrated below.
+
+| |Base (No Locality) |Washington DC Locality | San Francisco Locality |
+|---:|:---:|:---:|:---:|
+|**Grade 14**| $86,399|$107,325|$116,768|
+
+Here is a [GS Salary Calculator](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/2015/general-schedule-gs-salary-calculator/) if you would like to see how your locality affects compensation.
+
+Locality pay is a geographic-based percentage rate that reflects pay levels for non-federal workers in certain geographic areas as determined by [surveys conducted by the U.S. Bureau of Labor Statistics](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/fact-sheets/tabs/bls-data/). There are currently [34 locality pay areas](https://www.opm.gov/policy-data-oversight/pay-leave/salaries-wages/fact-sheets/tabs/locality-pay-areas/), which cover the lower 48 states and Washington, D.C., plus Alaska, Hawaii, and the U.S. territories and possessions.
+
+Each GS grade level contains a series of 10 steps which also have an impact on compensation. A new GS employee is usually hired at step 1 of the applicable GS grade. However, in special circumstances, agencies may authorize a higher step rate for a newly-appointed federal employee based on a [special need of the agency or superior qualifications of the prospective employee](https://www.opm.gov/policy-data-oversight/pay-leave/pay-administration/fact-sheets/superior-qualifications-and-special-needs-pay-setting-authority/).
+
+### Joining 18F
```

Comment:
@fureigh Talking to the team, they proposed moving the Joining 18F & Selecting Grade level sections to the top of this page.

I could go either way, leaning more toward moving to the top. If we are going to be directing applicants to this section from the application, they will probably be looking for information specific to applying first. Then if they need for information that is contained there, they can continue down to the rest.

What do you think?